aws-event-bridge/src/main/scala/akka/stream/alpakka/aws/eventbridge/javadsl/EventBridgePublisher.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.aws.eventbridge.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.stream.alpakka.aws.eventbridge.EventBridgePublishSettings
+import akka.stream.javadsl.{Flow, Keep, Sink}
+import akka.{Done, NotUsed}
+import software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient
+import software.amazon.awssdk.services.eventbridge.model._
+
+/**
+ * Java API
+ * Amazon EventBridge publisher factory.
+ */
+object EventBridgePublisher {
+
+  /**
+   * creates a [[akka.stream.javadsl.Flow Flow]] to publish messages to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def createFlow(settings: EventBridgePublishSettings,
+                 eventBridgeClient: EventBridgeAsyncClient): Flow[PutEventsRequestEntry, PutEventsResponse, NotUsed] =
+    akka.stream.alpakka.aws.eventbridge.scaladsl.EventBridgePublisher.flow(settings)(eventBridgeClient).asJava
+
+  /**
+   * creates a [[akka.stream.javadsl.Flow Flow]] to publish messages to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def createFlow(eventBridgeClient: EventBridgeAsyncClient): Flow[PutEventsRequestEntry, PutEventsResponse, NotUsed] =
+    akka.stream.alpakka.aws.eventbridge.scaladsl.EventBridgePublisher
+      .flow(EventBridgePublishSettings())(eventBridgeClient)
+      .asJava
+
+  /**
+   * creates a [[akka.stream.javadsl.Flow Flow]] to publish messages to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def createFlowSeq(
+      settings: EventBridgePublishSettings,
+      eventBridgeClient: EventBridgeAsyncClient
+  ): Flow[PutEventsRequestEntry, PutEventsResponse, NotUsed] =
+    akka.stream.alpakka.aws.eventbridge.scaladsl.EventBridgePublisher.flow(settings)(eventBridgeClient).asJava
+
+  /**
+   * creates a [[akka.stream.javadsl.Flow Flow]] to publish messages to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
```

Comment:
The `flowSeq` factory methods scaladocs should probably describe the difference between the standard methods.